linked_list/linked_list.py

```diff
@@ -23,13 +23,57 @@ def __init__(self):
     self.tail = None
 
   def add_to_tail(self, value):
-    pass
+    exists = self.head
+    while(exists):
+      exists = exists.next_node
+    exists = value
+    self.tail = value
+    return True
 
   def remove_head(self):
```

Comment:
For a list that has multiple nodes, you also want to set the new head (i.e. ```oldHead.get_next()```) of the linked list.